packages/design-system/src/styles/components/_Table.scss

```diff
@@ -1,13 +1,18 @@
 @import '../settings/index.scss';
 
+$table-header-background-color: $color-gray-lightest !default;
```

Comment:
These variables should be documented in the Guidance section of the Table docs. See [here](https://github.com/CMSgov/design-system/blob/master/packages/design-system-docs/src/pages/components/ChoiceList/_ChoiceList.docs.scss#L92) for an example